documentation/src/components/SupportedEnvironments.js

```diff
@@ -4,7 +4,7 @@ import Admonition from "@theme/Admonition";
 const SupportedEnvironments = () => {
   return (
     <Admonition type="info" title="Supported Environments">
-      Goose currently works on <strong>macOS</strong> and <strong>Linux</strong> systems and supports both <strong>ARM</strong> and <strong>x86</strong> architectures. If you'd like to request support for additional operating systems, please{" "}
+      The Goose CLI currently works on <strong>macOS</strong> and <strong>Linux</strong> systems and supports both <strong>ARM</strong> and <strong>x86</strong> architectures. However, the macOS GUI client does not currently work for x86, but will be supported soon. If you'd like to request support for additional operating systems, please{" "}
```

Comment:
```suggestion
      The Goose CLI currently works on <strong>macOS</strong> and <strong>Linux</strong> systems and supports both <strong>ARM</strong> and <strong>x86</strong> architectures. However, the macOS GUI client does not currently work for x86. If you'd like to request support for additional operating systems, please{" "}
```


legal tells me not to promise ppl things 😂